pkgs/tools/audio/audiowaveform/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, boost, gd, libsndfile, libmad, libid3tag }:
+
+stdenv.mkDerivation rec {
+  pname = "audiowaveform";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "bbc";
+    repo = "audiowaveform";
+    rev = version;
+    sha256 = "0k2s2f2hgq4pnjzfkgvjwgsflihmzdq7shicfjn0z2mzw4d1bvp2";
+  };
+
+  buildInputs = [ cmake boost gd libsndfile libmad libid3tag ];
+
+  cmakeFlags = [ "-DENABLE_TESTS=0" ];
```

Comment:
Is it possible to enable tests?

Reply:
Building the tests requires [Google Test](https://github.com/google/googletest) which I didn't see a package for so I left it out (could always be worked on in a follow-up PR). Their [homebrew file](https://github.com/bbc/homebrew-audiowaveform/blob/master/audiowaveform.rb) also leaves out tests.

Reply:
`gtest` package: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/gtest/default.nix

Reply:
Oh nice, added 👍